eslint-rules/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-uilib",
-  "version": "2.0.9",
+  "version": "2.1.0",
```

Comment:
I'm not sure if we should bump from 2.0.9 to 2.1.0 or to 2.0.10.. do you know if there's any logic for that?

Reply:
Base on [Semantic Versioning](https://semver.org/).
("MINOR version when you add functionality in a backwards compatible manner")